views/components/details/macro.njk

```diff
@@ -0,0 +1,6 @@
+{% macro details(title, isOpen = false, extraClasses) %}
```

Comment:
Added a common macro for this to abstract our some of the common styles, notably needing to wrap the contents in an extra `<div>` for IE support.